### PR TITLE
Substitutes docker in place of docker-compose for the build

### PR DESCRIPTION
**Pull request by JeremyGibson**
Seemed odd to have a `docker-compose build` step then a `docker tag` step, so this changes that.

Also adds a test_files directory with two dummy Dockerfiles for testing purposes.

Modifies the docs.

### CODE REVIEW

File: kubesae/image.py [hunk 1]
```diff
@@ -23,20 +23,22 @@ def generate_tag(c):
 
 
 @invoke.task(pre=[generate_tag])
-def build_image(c, tag=None):
+def build_image(c, tag=None, dockerfile=None):
     """Build Docker image using docker-compose.
```
Comment (dpoirier) [inline kubesae/image.py]: s/ using docker-compose// ?

File: invoke.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+app: test_app
+repository: TEST_REPO
+run:
+  pty: True
```
Comment (dpoirier) [inline invoke.yaml:4]: Is this new file related to the docker-compose change?

File: kubesae/image.py (hunk 1 above)
Comment (JeremyGibson) [reply kubesae/image.py]: Hrmmph. I thought I had changed that. Thanks

File: invoke.yaml (hunk 2 above)
Comment (JeremyGibson) [reply invoke.yaml:4]: It is needed to test any of the tasks that ask for the `app`. I added it  to the repo since it seemed convenient to have the ability to test tasks in the repo rather pulling it into an actual project.  But maybe that just means we should add real tests.

Comment (dpoirier) [reply invoke.yaml:4]: I'm fine with it so long as it's not packaged so it can't end up in the projects using kubesae.

**Review by dpoirier — APPROVED** [review]

File: invoke.yaml (hunk 2 above)
Comment (vkurup) [reply invoke.yaml:4]: Just to clarify... from my read of setup.py, nothing in the top level will be packaged, so this should be good, right? If so, I can merge this and release.

Comment (vkurup) [reply invoke.yaml:4]: Yes, from some testing, it looks like that is the case. I'm going to merge and release this so that I can use it with Pressweb.
